modules/eu_webtools_captcha/eu_webtools_captcha.module

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for eu_webtools_captcha module.
+ */
+
+use Drupal\oe_webtools_maps\Component\Render\JsonEncoded;
```

Comment:
This probably cannot depend on oe_webtools_maps
cf. https://github.com/openeuropa/oe_webtools/pull/140

Reply:
You're right, that dependency shouldn't be there. Replaced it with the default JSON class provided by Drupal